### PR TITLE
added "feed me" button and new icons

### PR DESCRIPTION
**Pull request by AmitSilber**

### CODE REVIEW

File: app/src/main/res/layout/content_choose_ingredients.xml [hunk 1]
```diff
@@ -22,14 +22,28 @@
...
         android:horizontalSpacing="10dp"
+        android:numColumns="auto_fit"
         android:stretchMode="columnWidth"
-        android:gravity="center"
-        />
+        android:verticalSpacing="10dp"
+        tools:layout_editor_absoluteX="0dp" />
+
+    <Button
+        android:id="@+id/button"
```
Comment (mfocshaner) [inline app/src/main/res/layout/content_choose_ingredients.xml]: change id to something indicative

File: app/src/main/res/layout/activity_choose_ingredients.xml [hunk 2]
```diff
@@ -20,7 +20,7 @@
 
     </android.support.design.widget.AppBarLayout>
 
-    <include layout="@layout/content_choose_ingredients" />
+    <!--<include layout="@layout/content_choose_ingredients" />-->
```
Comment (mfocshaner) [inline app/src/main/res/layout/activity_choose_ingredients.xml:23]: delete

File: app/src/main/res/layout/content_choose_ingredients.xml [hunk 3]
```diff
@@ -19,31 +19,35 @@
         app:layout_constraintTop_toTopOf="parent"
         app:layout_constraintVertical_bias="0.06" /-->
 
+    <ImageButton
+        android:id="@+id/imageButton"
```
Comment (mfocshaner) [inline app/src/main/res/layout/content_choose_ingredients.xml]: change to an indicative name

**Review by mfocshaner — APPROVED** [review]

File: app/src/main/res/layout/activity_choose_ingredients.xml (hunk 2 above)
Comment (AmitSilber) [reply app/src/main/res/layout/activity_choose_ingredients.xml:23]: i will delete you!

Comment (mfocshaner) [reply app/src/main/res/layout/activity_choose_ingredients.xml:23]: nooooooooo

**Pull request merged**